operators/src/processing/temporal_raster_aggregation/temporal_aggregation_operator.rs

```diff
@@ -75,6 +77,10 @@ impl RasterOperator for TemporalRasterAggregation {
         let initialized_operator = InitializedTemporalRasterAggregation {
             aggregation_type: self.params.aggregation,
             window: self.params.window,
+            window_reference: self
+                .params
+                .window_reference
+                .unwrap_or_else(|| TimeInstance::from_millis_unchecked(0)),
```

Comment:
we should add a constant for start of epoch.

Reply:
59341e43282f2e905b107ed01af3052be0bc683b